heudiconv/convert.py

```diff
@@ -1077,6 +1083,10 @@ def bvals_are_zero(bval_file: str) -> bool:
     True if all are all 0 or 5; False otherwise.
     """
 
+    # GE hyperband multi-echo containing diffusion info
+    if isinstance(bval_file, TraitListObject):
```

Comment:
note that we have `: str` types annotation for `bval_file`.  Here the premise is that it could be a list (like) as well.  We need to probably cast to regular list as soon as we get value there and adjust types annotations.

Also it would be great to gain some unittest on this aspect, otherwise this remains an untested and thus likely to break code.